app/views/products/_product.html.erb

```diff
@@ -0,0 +1,18 @@
+<tr>
+  <td><%= product.id %></td>
+  <td><%= image_tag product.image, size: Settings.cart.size_img %></td>
+  <td><%= product.name %></td>
+  <td><%= product.info %></td>
+  <td><%= product.category.name %></td>
+  <td class="text-left"><%= number_to_price(product.price) %></td>
```

Comment:
View giá bằng I18n em nhé

Reply:
function numer_to_price in ProductHelper đã dùng I18n